PatientRegistrationApp/src/components/GreetingPage.vue

```diff
@@ -1,82 +1,78 @@
 			</v-col>
+			<!--input buttons-->
 		</v-row>
-		<v-row align="center" justify="center">
-			
-			<v-btn-toggle group>
-				<v-btn 
-					class="ma-2"
-					outlined 
-					x-large 
-					color="primary"
-					height="13em"
-					width="17em"
-					@click="singleRegistration"
-				>
-				<div>
-					<v-icon size="7em">mdi-account</v-icon>
-					<span><br /><p style="font-size:1.2em" class="font-weight-regular"><br />Register myself</p></span>
-				</div>
-				</v-btn>
-				<v-tooltip bottom max-width="20%">
-					<template v-slot:activator="{on, attrs}">
+		<v-row justify="center">
+			<v-col cols="12">
+				<v-card class="d-flex justify-center flex-wrap" flat>
+					<v-card class="pa-2" flat>
 						<v-btn 
-							class="ma-2"
-							outlined 
+							:outlined="!isSingleRegistration" 
 							x-large 
 							color="primary"
-							v-bind="attrs"
-							v-on="on"
 							height="13em"
-							width="17em"
-							@click="householdRegistration"
-							>
-							<div>
-							<v-icon size="7em">mdi-account-group</v-icon>
-							<span><br /><p style="font-size:1.2em" class="font-weight-regular"><br />Register my household</p></span>
-							</div>
+							width="17.5em"
+							@click="singleRegistration"
+						>
+						<div>
+							<v-icon size="7em">mdi-account</v-icon>
+							<span><br /><p style="font-size:1.1em" class="font-weight-regular"><br />Register myself</p></span>
+						</div>
 						</v-btn>
-					</template>
-					<span>A household is composed of any people who occupy the same given housing unit. If you are planning to register your household, all members must currently be residing in the same address. If members of a household are residing in other housing units, it is necessary to register any given members individually. Max household registration of 20 registrants.</span>
-				</v-tooltip>
-			</v-btn-toggle>
-			
+					</v-card>
+					<v-card class="pa-2" flat>
+						<v-tooltip bottom max-width="20%">
+							<template v-slot:activator="{on, attrs}">
+								<v-btn 
+									:outlined="!isHouseholdRegistration" 
+									x-large 
+									color="primary"
+									v-bind="attrs"
+									v-on="on"
+									height="13em"
+									width="17.5em"
+									@click="householdRegistration"
+									>
+									<div>
+									<v-icon size="7em">mdi-account-group</v-icon>
+									<span><br /><p style="font-size:1.1em" class="font-weight-regular"><br />Register my household</p></span>
+									</div>
+								</v-btn>
+							</template>
+							<span>A household is composed of any people who occupy the same given housing unit. If you are planning to register your household, all members must currently be residing in the same address. If members of a household are residing in other housing units, it is necessary to register any given members individually. Max household registration of 20 registrants.</span>
```

Comment:
Maybe also make this a variable we import from a "CustomerSsettings" File or something? That way we are starting to build in easy functionality to customize things people might want to change (I imagine this might be one if a given client's definition of household is diff than ours.

Reply:
Yep!  Might as well start consolidating that stuff into one file for customer customization.

Reply:
Great!